2-Browsers/Week1/homework/ex6-gameOfLife/index.js

```diff
@@ -114,7 +128,18 @@ function createGame(context, numRows, numColumns) {
     // Apply the newly computed state to the cells
     forEachCell((cell) => {
       cell.alive = cell.nextAlive;
+      const {alive, nextAlive} = cell;
+      if(alive && nextAlive) {
+        cell.lifeTime += 1
```

Comment:
hint: you might also like this syntax
```suggestion
        cell.lifeTime++
```

Reply:
I think that this exercise wasn't ment to be uploaded.

>  You should NOT hand in these exercises, so make sure to not add it to the commit for your homework (have a look at the git add documentation here for all the options to exclude files in a commit).

https://github.com/HackYourHomework/Homework-class35/tree/master/2-Browsers/Week1#prep-exercises